dbms/src/Storages/DeltaMerge/tests/gtest_dm_minmax_index.cpp

```diff
@@ -2114,65 +2114,124 @@ try
         auto filter = createIn(attr("Nullable(Int64)"), {Field(static_cast<Int64>(3))});
         ASSERT_EQ(filter->roughCheck(0, 1, param)[0], RSResult::None);
     }
-    {
-        // make a not in filter, check not in (NULL)
-        auto filter = createNotIn(attr("Nullable(Int64)"), {Field()});
-        ASSERT_EQ(filter->roughCheck(0, 1, param)[0], RSResult::All);
-    }
-    {
-        // make a not in filter, check not in (NULL, 1)
-        auto filter = createNotIn(attr("Nullable(Int64)"), {Field(), Field(static_cast<Int64>(1))});
-        ASSERT_EQ(filter->roughCheck(0, 1, param)[0], RSResult::Some);
-    }
-    {
-        // make a not in filter, check not in (3)
-        auto filter = createNotIn(attr("Nullable(Int64)"), {Field(static_cast<Int64>(3))});
-        ASSERT_EQ(filter->roughCheck(0, 1, param)[0], RSResult::All);
-    }
```

Comment:
turn these into cases of `createNote(createIn(....))` ?

Reply:
Good idea! Fixed in https://github.com/pingcap/tiflash/pull/8212/commits/2fa3304b099fe1607d648814eb18774724febb33